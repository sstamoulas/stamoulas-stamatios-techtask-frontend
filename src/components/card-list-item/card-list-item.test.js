import React from 'react'
import { create } from 'react-test-renderer'
 
import CardListItem from './card-list-item.component'

describe('Card List Item Component', () => {
  let component
  let ingredient = 'Bread'
 
  beforeEach(() => { 
    component = create(<CardListItem ingredient={ingredient} />)
  })
 
  it('should render with given props', () => {
    expect(component.toJSON()).toMatchSnapshot()
  })
})
