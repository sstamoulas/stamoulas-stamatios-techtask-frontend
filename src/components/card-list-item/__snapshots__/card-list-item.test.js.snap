// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card List Item Component should render with given props 1`] = `
<li
  className="card-list-item"
>
  Bread
</li>
`;
